.github/workflows/publish-dev.yml

```diff
@@ -2,14 +2,14 @@ name: Publish dev images
 
 on:
   push:
-    branches: [ main, develop ]
+    branches: [ develop ]
```

Comment:
Since branch `main` deploys to PROD env, it's confusing to deploy `main` to DEV env. cc: @advcoder